frontend/ui/widget/dictquicklookup.lua

```diff
@@ -900,6 +915,15 @@ function DictQuickLookup:onCloseWidget()
         end
     end
 
+    -- Drop our ref from the static class member
+    for i = #DictQuickLookup.window_list, 1, -1 do
+        local window = DictQuickLookup.window_list[i]
+        -- We should only find a single match, but, better safe than sorry...
+        if window == self then
+            table.remove(DictQuickLookup.window_list, i)
```

Comment:
and `break` ?

Reply:
The comment was supposed to explain the lack of `break` ;o).

Reply:
But thinking back on it, this is `new`, not `show`, so, yeah, doesn't make sense ;o).

Reply:
Coward :)
(It feels even more safe now than before, now that it's `DictQuickLookup:init()` that add itself - and that probably can't add itselt twice.)

Reply:
Or does it? ;o).

`local dict = DQL:new{}` and then later `local d2 = DQL:new(dict)` is actually technically feasible (and both would add `dict` to the array, so there *could* be duplicates.

It's highly unlikely (and impossible with the current codebase), though ;).